### PR TITLE
Chore/testing

### PR DESCRIPTION
**Pull request by Yunnigg**

### CODE REVIEW

**Review by Vec7or — CHANGES_REQUESTED** [review]

File: pronto_mia/lib/ui/views/startup/startup_viewmodel.dart [hunk 1]
```diff
@@ -28,14 +27,13 @@ class StartUpViewModel extends BaseViewModel {
 
   @override
   Future<void> onFutureError(dynamic error, Object key) async {
-    await _errorService.handleError(
-        StartUpViewModel.contextIdentifier, modelError);
-    _errorMessage = _errorService.getErrorMessage(modelError);
+    super.onFutureError(error, key);
+    await _errorService.handleError(StartUpViewModel.contextIdentifier, error);
```
[the remark this answers is not in the document]
Comment (Yunnigg) [reply pronto_mia/lib/ui/views/startup/startup_viewmodel.dart:31]: modelError is null in this case

File: pronto_mia/lib/ui/views/pdf/pdf_viewmodel.dart [hunk 2]
```diff
@@ -36,7 +36,8 @@ class PdfViewModel extends FutureViewModel<SimpleFile> {
   @override
   Future<void> onError(dynamic error) async {
     super.onError(error);
-    await _errorService.handleError(PdfViewModel.contextIdentifier, modelError);
-    _errorMessage = _errorService.getErrorMessage(modelError);
+    await _errorService.handleError(PdfViewModel.contextIdentifier, error);
```
[the remark this answers is not in the document]
Comment (Yunnigg) [reply pronto_mia/lib/ui/views/pdf/pdf_viewmodel.dart:39]: modelError is null in this case

File: pronto_mia/test/setup/test_helpers.mocks.dart [hunk 3]
```diff
@@ -0,0 +1,116 @@
+import 'package:mockito/mockito.dart' as _i1;
```
[the remark this answers is not in the document]
Comment (Yunnigg) [reply pronto_mia/test/setup/test_helpers.mocks.dart:1]: yes

File: pronto_mia/test/viewmodels/deployment_plan_edit_viewmodel_test.dart [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+          ),
+        );
+        final deploymentPlanService = getAndRegisterMockDeploymentPlanService();
+        final dialogService = getAndRegisterMockDialogService();
+        deploymentPlanEditViewModel.formValueMap['description'] = 'bar';
+        deploymentPlanEditViewModel.formValueMap['availableFrom'] =
+            '2011-10-05T14:48:00.000Z';
+        deploymentPlanEditViewModel.formValueMap['availableUntil'] =
+            '2012-10-05T14:48:00.000Z';
+        deploymentPlanEditViewModel.formValueMap['pdfPath'] = 'test';
```
[the remark this answers is not in the document]
Comment (Yunnigg) [reply pronto_mia/test/viewmodels/deployment_plan_edit_viewmodel_test.dart:212]: pdf-path is assigned here. normally the corresponding view would fill the values from deploymentplan into the formfields.
I'm not sure what any of this has to do with the file fetching.

**Review by Vec7or — APPROVED** [review]

**Pull request merged**